pyproject.toml

```diff
@@ -1,3 +1,18 @@
 [build-system]
 requires = ["setuptools>=59", "wheel", "pyvex==9.2.33.dev0", "unicorn>=2.0.1"]
 build-backend = "setuptools.build_meta"
+
+[tool.ruff]
+extend-ignore = [
```

Comment:
I appreciate the comments, but does ruff support using names instead of magic numbers in its config file?

Reply:
If it does I'm un aware. I think ruff tries to borrow errors from other linters, like flake8, or pylint; so it uses error codes to make it easier to tell which linter and such.